src/security/defineUserAbility.ts

```diff
@@ -341,5 +341,12 @@ export default function defineUserAbility(user: User | Client): AppAbility {
     can(['create', 'read', 'update', 'delete'], 'Layer');
   }
 
+  /* ===
+    Creation / Access / Edition / Deletion of Archived Records
+  === */
+  if (userGlobalPermissions.includes(permissions.canSeeArchived)) {
```

Comment:
@MwanPygmay 
I know you didn't write that, but when taking a ticket already worked on by someone else, please check that the changes are relevant or not
this is not